core/nodejsActionBase/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:trusty-curl
+FROM buildpack-deps:xenial-curl
```

Comment:
It’s required to change to Xenia? Can we keep trusty?

Reply:
Xenial is necessary for the s390x port -- trusty does not have a supported port to s390x, I chose xenial as the next LTR.  I can add logic to make it conditional depending on platform -- Groovy templates as part of Gradle are a powerful thing, or we can start the inevitable move to Xenial.  Let me know which you prefer.